sdk/helper/keysutil/policy.go

```diff
@@ -493,8 +493,8 @@ type Policy struct {
 	deleted uint32
 
 	Name    string      `json:"name"`
-	Key     []byte      `json:"key,omitempty"`      // DEPRECATED
-	KeySize int         `json:"key_size,omitempty"` // For algorithms with variable key sizes
+	Key     []byte      `json:"key"`      // DEPRECATED
```

Comment:
Curious, why are we dropping the `omitempty` from these keys?

Reply:
That's a mistake.  When I was debugging I had added omitempty a bunch more places but though for a moment it was causing an issue, so removed them, but removed too many.